### PR TITLE
Determine if offline patch is available

### PR DESCRIPTION
**Pull request by guidograzioli**
AMW-337

### CODE REVIEW

**Review by rpelisse — APPROVED** [review]
LGTM!

File: roles/jws/tasks/apply_cp/apply_cp.yml [hunk 1]
```diff
@@ -6,14 +6,31 @@
...
   when: jws_offline_install
+  block:
+    - name: "Determine selected patch from supplied version: {{ jws_patch_version }}"
+      ansible.builtin.set_fact:
+        patch_version: "{{ jws_patch_version }}"
+        patch_bundle: "jws-{{ jws_patch_version }}{{ archive_file_suffix }}"
+
+    - name: "Check {{ jws_archive_repository }}/{{ patch_bundle }}"
+      ansible.builtin.stat:
+        src: "{{ jws_archive_repository }}/{{ patch_bundle }}"
```
Comment (jajik) [inline roles/jws/tasks/apply_cp/apply_cp.yml]: This does not look right, even linter is complaining...

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/stat_module.html

**Review by jajik — CHANGES_REQUESTED** [review]

File: roles/jws/tasks/apply_cp/apply_cp.yml (hunk 1 above)
Comment (guidograzioli) [reply roles/jws/tasks/apply_cp/apply_cp.yml]: yup

**Review by rpelisse — APPROVED** [review]
@guidograzioli LGTM!

**Pull request merged**